zulipterminal/ui_tools/views.py

```diff
@@ -895,18 +887,12 @@ def __init__(self, controller: Any, msg: Message) -> None:
         self.width = sum(max_widths)
         self.height = len(msg_info['Reactions'])+4 if msg['reactions'] else 5
 
-        self.log = urwid.SimpleListWalker(
-            [urwid.AttrWrap(
+        log = [urwid.AttrWrap(
```

Comment:
Since this is no longer a `...Walker`, let's name it more descriptively, as per my other comment.